AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -69,6 +69,11 @@ ba-computer.at,stoertebeker.com,inateck.com,biershop-bayern.de,nkd.com,4f.com.pl
 gfycat.com##.toast-notification--terms-and-privacy
 decathlon.com.br##.cookie-policies
 cam4.com##div[class^="CookieConsent"]
+radeberger-gruppe.de#$#.age-cookie-req { display: none!important; }
+radeberger-gruppe.de#$#body { overflow: visible!important; }
+radeberger-gruppe.de#$#.cookie-blur { filter: none!important; pointer-events: auto!important; transform: none!important; }
```

Comment:
It's better to change them to the rule like this `radeberger-gruppe.de#%#document.cookie = "dataPrivacy-acceptCookies=confirmed";`